src/game.cpp

```diff
@@ -4971,6 +4971,17 @@ void Game::updateCreatureWalkthrough(const Creature* creature)
 	}
 }
 
+void Game::updateKnownCreature(const Creature* creature)
```

Comment:
This sounds a bit misleading, as it is not a known creature, this just sends an update of a creature to spectators. Known creatures are cached and may not be in spectator-range of a player. See the comment I've left in Monster::setName.